migrations/20200214223200-my-script.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  async up(db, client) {
+    await db.collection('newCollection').insertOne({ _id: '1', key: 'test' });
+    await db.collection('newCollection').deleteOne({ _id: '1', key: 'test' });
+    // you can write some tests to verify everything is good and throw error if not.
```

Comment:
Not critical, but it looks like `newCollection` stays in the db forever eh, do we wanna keep this collection?

Reply:
I can add a statement to drop it

Reply:
It's up to you, I won't loose sleep over it :stuck_out_tongue: